github/ci/prow-deploy/files/jobs/kubevirt/containerdisks/containerdisks-presubmits.yaml

```diff
@@ -87,6 +87,43 @@ presubmits:
           privileged: true
       nodeSelector:
         type: bare-metal-external
+  - always_run: true
+    optional: false
+    annotations:
+      testgrid-create-test-group: "false"
+    cluster: prow-s390x-workloads
+    decorate: true
+    decoration_config:
+      grace_period: 5m0s
+      timeout: 1h0m0s
+    labels:
+      preset-podman-in-container-enabled: "true"
+      preset-kubevirtci-check-provision-env: "true"
```

Comment:
I don't think this is needed - it is just needed for the check-provision jobs from the kubevirtci repos.

Reply:
Alright, I will delete it and check again. A simpler setup is always preferable.